### PR TITLE
Split ezy examples into basic and advanced.

### PR DESCRIPTION
**Pull request by juanchopanza**
Basic examples require no neurom modules other than neurom.ezy.
Advanced examples require a few other neurom components. These will be
repackaged to make their inclusion more straight-forward.

### CODE REVIEW

**Comment by juanchopanza** [comment]
@liesbethvanherpe, @berthelm FYI I split up the ezy module examples. The "basic" one doesn't need any other neurom imports.

File: examples/ezy_advanced.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+from neurom.analysis import morphmath as mm
+from neurom.analysis import morphtree as mt
+import numpy as np
+
+
+def stats(data):
+    '''Dictionary with summary stats for data
+
+    Returns:
+        dicitonary with length, mean, sum, standard deviation,\
```
Comment (liesbethvanherpe) [inline examples/ezy_advanced.py]: *dictionary

File: examples/ezy_advanced.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    # get mean radius of segments
+    print('Mean radius of segments:',
+          np.mean([r for r in nrn.iter_segments(mm.segment_radius)]))
+
+    # Number of bifurcation points.
+    # This uses the more generic iter_neurites method, in which
+    # we can decide the type of iteration. Here we iterate over
+    # bifurcation points.
+    print('Number of bifurcation points:',
+          sum(1 for _ in nrn.iter_neurites(ibifurcation_point)))
```
Comment (liesbethvanherpe) [inline examples/ezy_advanced.py]: Can you simplify this sum expression too?

File: examples/ezy_advanced.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+    # Number of bifurcation points for apical dendrites
+    print('Number of bifurcation points (apical dendrites):',
+          sum(1 for _ in nrn.iter_neurites(ibifurcation_point,
```
Comment (liesbethvanherpe) [inline examples/ezy_advanced.py]: And this one?

File: examples/ezy_advanced.py (hunk 2 above)
Comment (juanchopanza) [inline examples/ezy_advanced.py]: No, this one sums 1 for each element, i.e. a transformation is required.

File: examples/ezy_advanced.py (hunk 1 above)
Comment (juanchopanza) [inline examples/ezy_advanced.py]: I will fix this one later on.

File: examples/ezy_advanced.py (hunk 2 above)
Comment (liesbethvanherpe) [inline examples/ezy_advanced.py]: OK, I see.

**Pull request merged**